modules/_labs/pagination/react/lib/Pages.tsx

```diff
@@ -0,0 +1,124 @@
+import {css, jsx} from '@emotion/core';
+import {IconButton} from '@workday/canvas-kit-react-button';
+import canvas from '@workday/canvas-kit-react-core';
+import _ from 'lodash';
+import React, {useRef} from 'react';
+
+import {PaginationAriaLabelsDefault} from './PaginationAriaLabels';
+
+/** @jsx jsx */
+interface PagesProps {
+  numPages: number;
+  currentPage: number;
+  onPageClick: (page: number) => void;
+  mobile: boolean;
+  ariaLabels: PaginationAriaLabelsDefault;
+}
+
+interface PaginationButtonProps {
+  page: number;
+  onPageClick: (page: number) => void;
+  active?: boolean;
+  ariaLabel: string;
+}
+
+const noPointerEvents = css({
+  pointerEvents: 'none',
+});
+
+const activeStyling = css(noPointerEvents, {
+  color: canvas.colors.frenchVanilla100,
+});
+
+const Ellipse = ({ariaLabel}: {ariaLabel: string}) => (
+  <IconButton
```

Comment:
I'm guessing `IconButton` is being used for the styling? We're disabling everything that makes it a button. Is there a better way to do this?

Reply:
okay I replaced it a span